src/selectors/entities/users.js

```diff
@@ -400,12 +400,10 @@ export const shouldShowTermsOfService: (GlobalState) => boolean = createSelector
     getConfig,
     getCurrentUser,
     getLicense,
-    getMyAcceptedTermsOfServiceData,
-    (config, user, license, myAcceptedTermsOfServiceData) => {
```

Comment:
Remove the `getMyAcceptedTermsOfServiceData` selector above if there are no usages.